src/pages/Boards/[id]/boardDetailContext.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState, useEffect } from 'react';
+import { useLocation } from 'umi';
+
+import { fetchBoard, fetchStatusList, fetchTasks } from './service';
+
+const BoardDetailContext = React.createContext({
+  dataBoard: {
+    task: {},
+    columns: {},
+    columnOrder: [],
+    title: '',
+    createdAt: '',
+    description: '',
+    members: [],
+  },
+  setDataBoard: null,
+  boardId: 0,
+});
+
+export const BoardDetailProvider = (props) => {
+  const location = useLocation();
+  const boardId = location.pathname.split('/')[2];
```

Comment:
use match router get param 
Example
const {
      dispatch,
      match: { params },
    } = this.props;
    if (get(params, 'id')) {
      dispatch({
        type: 'departmentAdd/GET_DETAILS',
        payload: get(params, 'id'),
      });
    }